auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java

```diff
@@ -248,8 +248,12 @@ private void startAuthMethodChoice() {
                     break;
                 case PhoneAuthProvider.PROVIDER_ID:
                     // Go directly to phone flow
+                    String phone = idpConfigs
+                            .get(0)
+                            .getParams()
+                            .getString(AuthUI.PHONE_VERIFICATION_DEFAULT_PHONE_EXTRA);
                     startActivityForResult(
-                            PhoneVerificationActivity.createIntent(getContext(), flowParams, null),
+                            PhoneVerificationActivity.createIntent(getContext(), flowParams, phone),
```

Comment:
This `createIntent` call uses `ExtraConstants.EXTRA_PHONE`.  We should unify the values between this and AuthUI.PHONE_VERIFICATION_DEFAULT_PHONE_EXTRA.

I'd suggest:

```java

class AuthUI {
  // ...
  public static final String EXTRA_DEFAULT_PHONE = ExtraConstants.EXTRA_PHONE;
  // ...
}
```

Reply:
That's reasonable, made your suggested change.